crates/bevy_ui/src/lib.rs

```diff
@@ -89,6 +91,8 @@ impl Plugin for UiPlugin {
             .register_type::<AlignSelf>()
             .register_type::<BackgroundColor>()
             .register_type::<CalculatedClip>()
+            .register_type::<TextLayoutInfo>()
+            .register_type::<TextFlags>()
```

Comment:
You need to add those conditionally.

To check your changes, use the `cargo check -p bevy_ui --no-default-features` command